libs/Zend/Db/Adapter/Mysqli.php

```diff
@@ -312,16 +312,64 @@ protected function _connect()
             }
         }
 
+        if($this->_config['usessl']) {
+            if(empty($this->_config['sslkey'])) {
+                $sslkey = null;
+            } else {
+                $sslkey = $this->_config['sslkey'];
+            }
```

Comment:
I would reformate this and the following if/else statements as it's shorter and easier to read:
```php
$sslkey = null;
if (!empty($this->_config['sslkey'])) {
    $sslkey = $this->_config['sslkey'];
}
```